package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@elastic/package-registry",
+  "version": "0.2.0",
+  "description": "Elastic Package Registry (EPR)",
+  "repository": "git@github.com:elastic/package-registry.git",
+  "author": " Nicolas Ruflin <spam@ruflin.com>",
+  "license": "Elastic-License",
+  "types": "./types/types.d.ts",
+  "main": "./types/types.js"
```

Comment:
I see we have the path specified here. So if we ever move it around in the repo, we can just update it here 👍